examples/android/helloworld/app/build.gradle

```diff
@@ -0,0 +1,53 @@
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion 26
+    defaultConfig {
+        applicationId "io.grpc.android.cpp.helloworldexample"
+        minSdkVersion 14
+        targetSdkVersion 26
+        versionCode 1
+        versionName "1.0"
+        testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
+        externalNativeBuild {
+            cmake {
+                cppFlags "-std=c++14 -frtti -fexceptions"
+                arguments '-DANDROID_STL=c++_static'
+                arguments '-DRUN_HAVE_POSIX_REGEX=0'
+                arguments '-DRUN_HAVE_STD_REGEX=0'
+                arguments '-DHAVE_STEADY_CLOCK=0'
```

Comment:
Are you sure about this one? I get: 

> Manually-specified variables were not used by the project:
>
> HAVE_STEADY_CLOCK

It doesn't prevent me from building for Android (running on Arch Linux), but I am trying to use your changes to build for iOS on a Macbook Pro, and there I get an error:

> CMake Error: TRY_RUN() invoked in cross-compiling mode, please set the following cache variables appropriately:
>
> RUN_HAVE_STEADY_CLOCK

Using `-DRUN_HAVE_STEADY_CLOCK` instead solves the problem.

Reply:
You're right. Switched to `-DRUN_HEAD_STEADY_CLOCK`